lib/actions/api.js

```diff
@@ -263,7 +263,11 @@ export function createQueryAction(
       }
     }
 
-    const url = makeApiUrl(config, endpoint, options)
+    const url = options?.url
+      ? `${config?.api?.host}${config?.api?.port ? ':' + config.api.port : ''}${
```

Comment:
The host/port of the URL is repeated code with `makeApiUrl` and should be extracted, and why are we introducing a new `options.url` that duplicates the `endpoint` parameter?

Reply:
The problem is that makeApiUrl appends `router/default` which we can't have here. I thought about adapting it to handle this new use case but once we're fully on graphQL it'll be deleted anyway!

Reply:
I still think a function that deals with this portion: `${config?.api?.host}${config?.api?.port ? ':' + config.api.port : ''}${<arg>}` would be helpful.

Reply:
I've removed `makeApiUrl`